content/docs/dvclive/get-started.md

```diff
@@ -20,11 +20,38 @@ See [`Live()`](/doc/dvclive/api-reference/live) for details.
 
 ### Log data
 
+<toggle>
+<tab title="Scalars">
+
 ```python
 live.log("acc", 0.9)
```

Comment:
In this case the tabs contents are pretty short so I think it's definitely better to flatten this to avoid hiding important content.